app/controllers/application_controller.rb

```diff
@@ -1,3 +1,5 @@
+require "google/cloud/bigquery"
```

Comment:
Do we need this require statement here?  I though including it in the Gemfile was sufficient.

Reply:
Whoops, should have just done it in the gemfile...

Reply:
all set now